src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs

```diff
@@ -62,7 +62,7 @@ public ICollection<string> Exclude
         [Option(typeof(NuGetCommand), "PackageCommandIncludeReferencedProjects")]
         public bool IncludeReferencedProjects { get; set; }
 
-        [Option(typeof(NuGetCommand), "PackageCommandPropertiesDescription")]
+        [Option(typeof(NuGetCommand), "PackageCommandPropertiesDescription", AltName = "p")]
```

Comment:
Do we have a test for this or did you catch it yourself?

Reply:
Basically if you caught it yourself, do you mind adding a test for that if the infrastructure exists.

Reply:
I added the AltName because a number of the existing unit tests use `-p` as shorthand for `-Properties`.  Once I added '-PackagesDirectory', `-p` became ambiguous and all of those tests failed.

I removed the AltName and re-ran `build.ps1`; from the output, it looks like everything passed this time?  It's been 7 months; I don't know if someone cleaned up the unit tests since then.

I tried checking out the old version (based on `dev` from 2018-10) to verify this, but it won't build; I get this error:

```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.TargetFrameworkInference.targets(1
12,5): error : The current .NET SDK does not support targeting .NET Core 2.0.  Either target .NET Core 1.1 or lower, or use a version of the .NET SDK
that supports .NET Core 2.0. [N:\NuGet.Client.2018Oct\test\TestExtensions\GenerateLicenseList\GenerateLicenseList.csproj]
Invoke-BuildStep : Failed - Running Restore for VS 15.0
At N:\NuGet.Client.2018Oct\build.ps1:118 char:1
+ Invoke-BuildStep 'Running Restore for VS 15.0' {
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Invoke-BuildStep

Error-Log : [11:57:55 +8]       [FAILED +00:00:07] Running Restore for VS 15.0
At N:\NuGet.Client.2018Oct\build\common.ps1:142 char:17
+ ...             Error-Log "[FAILED +$(Format-ElapsedTime $sw.Elapsed)] $B ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Error-Log
```

I'm fairly certain I have the .NET Core 2.0 SDK installed, so I don't know why it's failing now.

Reply:
No one has removed those tests, so the scenario is covered.
Quick search suggests that the tests are still there.
Not sure why the script is failing but it seems to be picking the wrong sdk version. 
I wouldn't spend time and effort on it though. 
I will resolve this conversation, as you've answered my question.